R/estimate_CUMINC.R

```diff
@@ -0,0 +1,89 @@
+#' Competing Events Cumulative Incidence
+#'
+#' @param CNSR Column name indicating the outcome and censoring statuses.
+#' Column must be a factor and the first level indicates censoring, the
+#' next level is the outcome of interest, and the remaining levels are the
+#' competing events.
+#' @param conf.int Confidence internal level. Default is 0.95.
+#' @param ... Additional argument passed to `tidycmprsk::cuminc()`
+#' @inheritParams estimate_KM
+#' @inheritParams visr
+#' @inheritParams add_CI.ggsurvfit
+#' @inheritParams add_risktable.ggsurvfit
+#' @inheritParams get_risktable
+#'
+#' @export
+#'
+#' @examples
+#' estimate_cuminc(
+#'   tidycmprsk::trial,
+#'   strata = "trt",
+#'   CNSR = "death_cr",
+#'   AVAL = "ttdeath"
+#' ) %>%
+#'   visr() %>%
+#'   add_CI() %>%
+#'   add_risktable(statlist = c("n.risk", "cumulative.event"))
+
+estimate_cuminc <- function(data
+                            ,strata = NULL
+                            ,CNSR
```

Comment:
Can we do data, ... CNSR = "CNSR", AVAL = "AVAL", ..?

Reply:
Done! ✔️ I had originally not added them because the definitions are not the standard ADAM (I don't think there is a standard for competing event outcome definitions).

Reply:
I think competing events in ADaM will still follow the standard time-to-event structure where CNSR has multiple levels and EVNTDESC is describing the competing events.